modules/developer_manual/pages/webdav_api/trashbin.adoc

```diff
@@ -0,0 +1,229 @@
+= Trashbin API
+:toc: right
+:toclevels: 1
+
+== Introduction
+
+The trash bin API provide support for managing files in ownCloud’s trash bin.
+Specifically, it provides the following functionality:
+
+* xref:list-files[List files in the trash bin]
+* xref:delete-files[Delete files permanently from the trash bin]
+* xref:restore-deleted-file[Restore deleted files to the trash bin]
+
+== List Files
+
+List all files in the trash bin
+
+* Path: `remote.php/dav/trash-bin/<username>`
+* Method: `PROPFIND`
+
+=== Request Parameters
+
+[cols=",,",options="header",]
+|===
+|Attribute 
+|Type 
+|Description
+
+|`username` 
+|string 
+|The username of the user to list files for.
+|===
+
+=== Request Headers
+
+[cols=",,",options="header",]
+|===
+|Header
+|Type
+|Description
+
+|`Depth`
+|int
+a|The maximum level to descend to, when retrieving a listing of the files in the user's trash bin.
```

Comment:
```suggestion
|The maximum level to descend to, when retrieving a listing of the files in the user's trash bin.
```

Reply:
The `a` that precedes the cell content is required for the admonition to render correctly.